viewer/packages/pointclouds/src/CognitePointCloudModel.ts

```diff
@@ -55,7 +55,7 @@ export class CognitePointCloudModel implements CogniteModelBase {
   /**
    * Used to clean up memory.
    */
-  dispose(): void {}
+  dispose(): void { }
```

Comment:
Seems like you have to run `yarn lint --fix` because CI is failing and this line shouldn't be changed.

Reply:
Fixed 👍